### PR TITLE
Unique Heretic Ascension Announcements

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

All the audio work done by @Flleeppyy, coding done by me (Absolucy), with feedback and initial ideas from @aberrantQuesrist and @KittyNoodle

[Preview of all sounds](https://www.youtube.com/watch?v=v5vBTgRPQKU)

## Why It's Good For The Game

Adds extra flavor and spice to the game for important events, much better than the overused generic "space-time anomalies" announcement.

## Changelog
:cl: Flleeppyy, Absolucy
sound: Added unique announcement jingles to each heretic path's ascension, replacing the old "space-time anomalies detected" announcement.
/:cl:

### CODE REVIEW

**Comment by Rex9001** [comment]
This sounds really good, I love the music. One note would be Im afraid of the overlap of the lunatic gain track, but honestly just remove the greet music for that antag in favor of this dope track.

**Comment by tralezab** [comment]
All of these are fantastic! Super great quality and some stand out as exceptionally good and fitting. Flesh is perfect.

To give you some usable feedback, some of them definitely feel a lot more like a fitting ascension sound than others. Cosmic for example really giving me space ambience a lot more than a heretic ascension lmao. And then you have the one that has breakcore in the background which made me laugh. Obviously, none of these comments mean the pr is anywhere close to problematic. Let's get dis MERGED

**Comment by Sadboysuss** [comment]
badass!! I love the sounds, I would love to see these sounds play for the heretic when they pick their path and then play when the heretic ascends so you can hear it more ingame!

**Comment by ViktorKoL** [comment]
Beat me to it! I even had a theme ready, the only thing I was waiting for was deciding whether I wanna make a unique one for each path 😄 

I would perhaps criticize how laid back some of these feel, they don't exactly always give the vibe of "you are about to die". But they are neat regardless

**Comment by Absolucy** [comment]
aberra and noodle were the ones that came up with the initial "ideas" for each one - i asked them due to the fact they're very familiar with both ss13 and cultist simulator.

here's the ideas that were used:

> common between them all?: Sending a chill down someone's spine?
> 
> **Flesh**- Changing something, twisting into a horrible creature. Looking for a disturbing sound of tearing meat.
> **Knock**- Opening a gateway that never should have been. Eldritch and chilling.
> **Cosmos**- Summoning of a godlike entity to serve you. Ethereal and majestic. 
> **Blade**- Triumphantly climbing above a path of combat. Proud and bold. 
> **Ash**- Burning EVERYTHING. All will fall. Hateful and frantic, like a flame.
> **Void**- An endless storm of winter, cold and sad. Calm but uneasy. 
> **Rust**- Creation and decay, conversion. Perhaps utilizing the sound of scraping metal somehow?
> **Moon**- Insanity, slowly unraveling. Maybe include laughter and crying alongside quiet piano? would be more of an intense drumbeat, maybe some cymbals.

**Review by LemonInTheDark — COMMENTED** [review]
~5.2mb isn't nothing but it's not so much for the size to be a problem here.
We're never adding a new heretic path huh

**Comment by tralezab** [comment]
> We're never adding a new heretic path huh

If we never added another we still have quite a lot and all of them are stuffed full of content, I wouldn't cry too badly considering how long heretic sat at just 3 paths

**Comment by san7890** [comment]
if we're truly concerned about the newly found talent-debt to maintain this really cool feature i think it might be nice to see if we could have a "generic" ascension sound that just ties into the antagonist as a whole just so it can be there for unimplemented soundtracks. it might sound lame but i don't see much harm in the concept of a placeholder while we still have the original sound artists/designers in one place.

if the original people can't make it work, then we shouldn't hold up the PR but i'm mentioning it now if it can be though of as a genuine concern.

File: code/modules/antagonists/heretic/heretic_antag.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 	if(give_objectives)
 		forge_primary_objectives()
 
-	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ecult_op.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)//subject to change
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/heretic/heretic_gain.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)//subject to change
```
Comment (san7890) [inline code/modules/antagonists/heretic/heretic_antag.dm:205]:
```suggestion
	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/heretic/heretic_gain.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
```

File: code/modules/antagonists/heretic/heretic_monsters.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 /datum/antagonist/heretic_monster/on_gain()
 	. = ..()
-	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ecult_op.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)//subject to change
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/heretic/heretic_gain.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)//subject to change
```
Comment (san7890) [inline code/modules/antagonists/heretic/heretic_monsters.dm:16]:
```suggestion
	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/heretic/heretic_gain.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
```

**Review by san7890 — COMMENTED** [review]
i have no idea what the old `//subject to change` comment means here but assuming that it's referring to using `ecult_op` it's even less relevant. greatly appreciated to get this cludge removed now

**Comment by Absolucy** [comment]
I mean, heretic pathing is currently spaghetti anyways, you need a table the size of the moon in order to properly understand it, I doubt anyone is going to be masochistic enough to make another path anytime soon.

And if someone else wants to, tbh, can contact me or chen and we might try.

**Comment by Ghommie** [comment]
Maybe we could add new heretic paths to the restricted content, like for humanoid species, just to be sure, who knows...

**Comment by flleeppyy** [comment]
YIPPEEE!!!!

**Comment by san7890** [comment]
> Maybe we could add new heretic paths to the restricted content, like for humanoid species, just to be sure, who knows...

ideally it should have been de facto restricted by now because we blow a gasket every time someone adds a new path without fixing the issues that paths currently have but ehhh

**Pull request merged**